apps/api/core/logs.py

```
@@ -2,6 +1,92 @@
+_allowed_log_levels = {
+    "DEBUG": logging.DEBUG,
+    "INFO": logging.INFO,
+    "WARNING": logging.WARNING,
+    "ERROR": logging.ERROR,
+    "CRITICAL": logging.CRITICAL,
+}
+LOG_LEVEL = _allowed_log_levels.get(_log_level_str, logging.INFO)
 
-print  # facade
+class CustomJsonFormatter(jsonlogger.JsonFormatter):
+    def add_fields(self, log_record, record, message_dict):
```

Ruff (0.11.9)

Missing return type annotation for public function `add_fields`

Add return type annotation: `None`

(ANN201)